code/datums/outfit/civilian.dm

```diff
@@ -669,6 +669,7 @@
 /datum/outfit/iaa/post_equip(var/mob/living/carbon/human/H)
 	..()
 	H.put_in_hands(new /obj/item/weapon/storage/briefcase/centcomm(H))
+	H.set_light(1, 3, "#006400") //Green
```

Comment:
Why not just do this with a glowing badge accessory instead?

Reply:
It's funnier without reliance to accessories